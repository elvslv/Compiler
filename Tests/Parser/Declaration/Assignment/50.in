type my_int = integer;
var a: array[1..10] of my_int;
begin
	a[1] := 12345;
end.