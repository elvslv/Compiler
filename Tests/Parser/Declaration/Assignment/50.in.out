type MY_INT = INTEGER;
var A : array [const 1..const 10] of MY_INT;
:=       
|        
|___[]   
|   |    
|   |___A
|   |    
|   |___1
|        
|___12345
